v2/tools/scripts/fetch.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -e
```

Comment:
can you please make sure these bash files are passing https://github.com/koalaman/shellcheck and ok to do a follow up to add the [shellcheck gh action](https://github.com/mongodb/mongodb-atlas-cli/blob/master/.github/workflows/shellcheck.yml)?